### PR TITLE
Update tracex installer.

### PR DESCRIPTION
**Pull request by bochen-1**

### CODE REVIEW

File: installer/tracex_installer_release.iss [hunk 1]
```diff
@@ -55,7 +55,7 @@ Source: "src\MsixPackageProject\TraceFiles\demo_threadx.trx"; DestDir: "{app}/Tr
 
 [Icons]
 Name: "{group}\TraceX 6.4\TraceX"; Filename: "{app}\TraceX.exe"
-Name: "{group}\TraceX 6.4\TraceX User's Guide"; Filename: "https://aka.ms/azrtos-tracex-user-guide"
+Name: "{group}\TraceX 6.4\TraceX User's Guide"; Filename: "https://github.com/eclipse-threadx/rtos-docs"
```
Comment (wickste) [inline installer/tracex_installer_release.iss:58]: I think this point directly to:
https://github.com/eclipse-threadx/rtos-docs/blob/main/rtos-docs/tracex/overview-tracex.md

File: src/TraceX/TraceXView.xaml.cs [hunk 2]
```diff
@@ -1157,7 +1157,7 @@ private void OnExit(object sender, RoutedEventArgs e)
         // Handler for Help -> Manual.
         private void OnOpenManual(object sender, RoutedEventArgs e)
         {
-            string url = "https://docs.microsoft.com/en-us/azure/rtos/tracex/";
+            string url = "https://github.com/eclipse-threadx/rtos-docs";
```
Comment (wickste) [inline src/TraceX/TraceXView.xaml.cs:1160]: I think this should point directly to:
https://github.com/eclipse-threadx/rtos-docs/blob/main/rtos-docs/tracex/overview-tracex.md

**Review by wickste — COMMENTED** [review]
Please update per comments

File: src/MsixPackageProject/Package.appxmanifest [hunk 3]
```diff
@@ -9,13 +9,13 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Name="Microsoft.AzureRTOSTraceX"
-    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
+    Name="Eclipse.EclipseThreadXTraceX"
+    Publisher="CN=Eclipse Foundation, O=Eclipse Foundation, L=Ottawa, S=Ontario, C=CA"
```
Comment (wickste) [inline src/MsixPackageProject/Package.appxmanifest:13]: I believe changing the publisher identity in the package.appxmanifest will prevent the store package (.msix) from being installable. It needs to match the certificate.

File: installer/tracex_installer_release.iss (hunk 1 above)
Comment (bo-ms) [reply installer/tracex_installer_release.iss]: updated

File: src/MsixPackageProject/Package.appxmanifest (hunk 3 above)
Comment (bo-ms) [reply src/MsixPackageProject/Package.appxmanifest]: reverted the publisher

File: src/TraceX/TraceXView.xaml.cs (hunk 2 above)
Comment (bo-ms) [reply src/TraceX/TraceXView.xaml.cs]: updated

File: src/MsixPackageProject/Package.appxmanifest [hunk 4]
```diff
@@ -9,13 +9,13 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Name="Microsoft.AzureRTOSTraceX"
+    Name="Eclipse.EclipseThreadXTraceX"
```
[the remark this answers is not in the document]
Comment (bo-ms) [reply src/MsixPackageProject/Package.appxmanifest]: reverted

**Review by wickste — APPROVED** [review]

**Pull request merged**